tests/cli/test_cli.py

```diff
@@ -101,14 +101,22 @@ def current_user():
     return user_id, user_name
 
 
-def create_user(context, username, password='codalab'):
+def create_user(context, username, password='codalab', disk_quota='100g'):
     # Currently there isn't a method for creating a user with the CLI. Use CodaLabManager instead.
     manager = CodaLabManager()
     model = manager.model()
 
     # Creates a user without going through the full sign-up process
     model.add_user(
```

Comment:
You could simplify this PR (and avoid the need to add new arguments to `add_user` that are only used in tests) by just adding:

```
model.default_user_info['disk_quota'] = disk_quota
```

Reply:
yeah that's a lot simpler